iota/commands/extended/utils.py

```diff
@@ -27,111 +24,106 @@ def find_transaction_objects(adapter, **kwargs):
     hashes = ft_response['hashes']
 
     if hashes:
-      gt_response = GetTrytesCommand(adapter)(hashes=hashes)
+        gt_response = GetTrytesCommand(adapter)(hashes=hashes)
 
-      return list(map(
-        Transaction.from_tryte_string,
-        gt_response.get('trytes') or [],
-      )) # type: List[Transaction]
+        return list(map(
+            Transaction.from_tryte_string,
+            gt_response.get('trytes') or []
+        ))  # type: List[Transaction]
 
     return []
 
 
 def iter_used_addresses(adapter, seed, start):
-  # type: (BaseAdapter, Seed, int) -> Generator[Tuple[Address, List[TransactionHash]]]
-  """
-  Scans the Tangle for used addresses.
+    # type: (BaseAdapter, Seed, int)
+    #    -> Generator[Tuple[Address, List[TransactionHash]]]
```

Comment:
This does not work, unfortunately; the type hint has to all be on the same line.

If the type hint is too long, then it can be integrated into the function signature, like this:

```python
def iter_used_addresses(
        adapter,   # type: BaseAdapter
        seed,  # type: Seed
        start,  # type: int
):
    # type: (...) -> Generator[Tuple[Address, List[TransactionHash]], None, None]
```

(I also got the `Generator` type hint wrong; it actually requires 3 values, not 1 — the other two indicate what values can be passed in, and what value is returned; see [PEP-484](https://www.python.org/dev/peps/pep-0484/#annotating-generator-functions-and-coroutines) for details)